docs/develop/03_workshop/04_attester/03_ctype.md

```diff
@@ -14,75 +14,86 @@ import GenerateCtype from '!!raw-loader!@site/code_examples/sdk_examples/src/wor
+For example, a traffic authority only issues driver's licenses (A CType for driver's license), not trade register excerpts.
+
+Since CTypes enable interoperability between Attesters, using existing CTypes rather than creating new ones is highly recommended.
+However, this workshop creates a new CType to show the process.
+
 Creating CTypes requires an account and a full DID.
 Make sure your account holds KILT tokens so that you can pay the fees for creating a CType.
 
-For example, a very basic CType for a driver's license could look like this:
+For example, a basic CType for a driver's license could look like this:
 
 <CodeBlock className="language-json">
   {Ctype}
 </CodeBlock>
 
-Let's have a look at these attributes.
+The CType has the following attributes:
 
 | Key          | Value                                                                                                                                                               |
 | -------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `$id`        | The KILT id of this CType. It is the most important property as it represents the **digital footprint** of the CType.                                               |
+| `$id`        | The KILT id of this CType. It's the most important property as it represents the **digital footprint** of the CType.                                               |
 | `$schema`    | A reference to the meta-schema describing what a CType may look like. There are two versions.                                                              |
 | `title`      | The title of the CType.                                                                                                                                             |
 | `properties` | The properties that a claim conforming to this CType may have.                                                                                                      |
 | `type` | Type is an object for all CTypes.                                                                                                  |
-| `additionalProperties` | The default is set to false. This restricts unwanted properties in a claim.                                                                                                      |
+| `additionalProperties` | The default is false. This restricts unwanted properties in a claim.                                                                                                      |
 
 A CType is stored on the KILT blockchain.
 
-In a real-life setup, a user would simply retrieve an existing CType from the chain or a repository of CTypes for example via a Credential Registry's REST API.
+<!-- TODO: What is a Credential Registry's REST API? -->
```

Comment:
What is a Credential Registry's REST API?

Reply:
@ntn-x2 Yes, but what is it? Where can I find more about it? What options are available etc etc?

Reply:
I think @Dudleyneedham, who was the author of that section, might have more background info.

Reply:
Yes, I think we were describing the service mentioned above. We can remove the REST API.